src/Umbraco.Infrastructure/Persistence/Repositories/Implement/MacroRepository.cs

```diff
@@ -77,7 +77,7 @@ public IMacro GetByAlias(string alias)
 
         public IEnumerable<IMacro> GetAllByAlias(string[] aliases)
         {
-            if (aliases.Any() == false)
+            if (aliases.Any() is false)
             {
                 return base.GetMany();
             }
```

Comment:
@Zeegaan shouldn't it just use `Length` here since the aliases is `string[]`.

for example:

```
if (aliases?.Length == 0)
{
   
}
```

As far I know `Any()` check first item if collection, but with `Array`, `List`, `ICollection` etc,  `Length` or `Count` property already know the size of the collection.

Reply:
Hey @bjarnef, thanks for your comment, i had to check how this works 👀  When it is an array, Any() will also use the Count property like this:
![image](https://user-images.githubusercontent.com/70372949/159218631-4daabefc-60db-4900-94c2-52316c90d98f.png)
And because of that, i think its more readable if we use `Any()` instead of `Length`